.buildkite/upload_artifacts.py

```diff
@@ -203,7 +203,12 @@ def upload_artifacts():
             )
         blob.upload_from_filename(filename=file_data.get("file_location"))
         blob.make_public()
-        file_data.update({"media_url": blob.media_link})
+        file_data.update(
+            {
+                "size_mb": os.path.getsize(file_data.get("file_location")) / 1048576.0,
+                "media_url": blob.media_link,
+            }
+        )
```

Comment:
@micahscopes this PR seems to have changes in it which are unrelated to the work. Looks like there was some kind of rebase or merge issue.

Reply:
oh, maybe not. I might have been looking at the diff for an older commit